cmd/commands/root.go

```diff
@@ -58,30 +67,76 @@ var RootCmd = &cobra.Command{
 }
 
 func init() {
-	// Define your flags and configuration settings.
-	RootCmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "kubernetes config path")
+	// Initialize client configuration with all standard kubectl flags first
+	clientConfig = defaultClientConfig(RootCmd.PersistentFlags())
 	RootCmd.PersistentFlags().StringVar(&operatorNamespace, "operator-namespace", "", "Kubernetes namespace where rook operator is running")
-	RootCmd.PersistentFlags().StringVarP(&cephClusterNamespace, "namespace", "n", "rook-ceph", "Kubernetes namespace where CephCluster is created")
-	RootCmd.PersistentFlags().StringVar(&kubeContext, "context", "", "Kubernetes context to use")
+
+	// Set default ceph cluster namespace
+	cephClusterNamespace = "rook-ceph"
```

Comment:
Seems like the namespace would be assigned above on line 59, so assigning this default isn't needed?